### PR TITLE
Updated AKV role authorization to include proper conditional

### PR DESCRIPTION
**Pull request by jaswitze**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by jaswitze** [comment]
@bmoore-msft made the changes discussed earlier.

File: demos/sports-analytics-architecture/main.bicep [hunk 1]
```diff
@@ -14,18 +14,18 @@ Name of the Azure Data Lake Storage Gen2 storage account. Storage account name r
 ''')
 @minLength(3)
 @maxLength(24)
-param azureDataLakeStoreAccountName string = 'stor${uniqueString(resourceGroup().id)}'
```
Comment (bmoore-msft) [inline demos/sports-analytics-architecture/main.bicep:17]: Let's leave the defaultValues in so it's easier to deploy...

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by jaswitze** [comment]
@bmoore-msft made the changes requested.

File: demos/sports-analytics-architecture/main.bicep (hunk 1 above)
Comment (jaswitze) [reply demos/sports-analytics-architecture/main.bicep:17]: Updated.

File: demos/sports-analytics-architecture/azuredeploy.parameters.json [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+      "azureADObjectID": {
+        "value": "GEN-AZUREAD-OBJECTID"
+      },
+      "azureDataLakeStoreAccountName": {
+        "value": "GEN-UNIQUE"
+      },
+      "azureDataFactoryName": {
+        "value": "GEN-UNIQUE"
+      },
+      "azureDatabricksName": {
```
[the remark this answers is not in the document]
Comment (jaswitze) [reply demos/sports-analytics-architecture/azuredeploy.parameters.json]: Updated

**Pull request merged**